eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -179,6 +190,7 @@ private Collection<Dependency> deps() throws IOException {
             deps.add(one);
             tojo.set(AssembleMojo.ATTR_JAR, coords);
         }
+        checkConflicts(deps, !this.ignoreVersionConflicts);
```

Comment:
@mximp what's the point of passing this boolean value as function argument, if it's available as attribute in this very class?